deploy/modules/statistics/statistics.tpl

```diff
@@ -1,36 +1,23 @@
 #!/bin/bash -x
 set -e
 
-statistics_bucket_name="04274532-55f0-11ed-bdc3-0242ac120002"
+if [[ "$(aws --version)" == *"aws-cli"* ]]
```

Comment:
Usually "command -v <command>" is used

Reply:
Can you elaborate ?

Reply:
Not a critical issue. I guess you wanted to verify that aws cli is installed. You don't care what version it is or whether the content is "aws-cli". That's why I suggested another option which only verifies the aws command exists on the system.